core/services/keeper/orm.go

```diff
@@ -164,7 +164,7 @@ func (korm ORM) CreateEthTransactionForUpkeep(ctx context.Context, upkeep Upkeep
 		upkeep.ExecuteGas+gasBuffer,
 	).Scan(&etx.ID)
 	if err != nil {
-		return etx, errors.Wrap(err, "keeper failed to insert eth_tx")
+		return etx, errors.Wrap(err, "keeper failed to insert eth_tx, node is likely out of funding")
```

Comment:
Since Keeper is now using the exact same logic, do you think it makes sense to pull this common InsertTx code into bulletprooftxmanager because it's called in 2-3 other places?

Reply:
@samsondav yeah that's a good point - I forgot we had discussed that. I'll refactor.